### PR TITLE
Adding details to README for ACA data analysis.

### PR DESCRIPTION
**Pull request by eco-ben**
Detailed data structures for ACA and Wave data, ACA crs and the intent of each numbered script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,19 +29,25 @@ MPA_DATA_DIR = "path to GBR data"  # location of GBR datasets
...
 
-`zones` holds GBRMPA zone layers in geojson format.
+MPA_DATA_DIR : contains raster data at whole-GBR and GBRMPA-management-region scales.
+- `zones` holds GBRMPA zone layers in geojson format.
+- `features` holds the GBRMPA GBR-wide feature set.
+
+ACA_DATA_DIR : contains raster and vector data at whole-GBR scale.
 
-`features` holds the GBRMPA GBR-wide feature set.
+WAVE_DATA_DIR : contains wave NetCDF data at GBRMPA-management-region scale.
```
Comment (ConnectedSystems) [inline README.md]: Something like this?

```suggestion
WAVE_DATA_DIR : contains wave data in NetCDF format at the scale of GBRMPA management regions.
```

File: README.md [hunk 2]
```diff
@@ -58,12 +64,33 @@ MPA_DATA_DIR
...
+│   └───Vector data
+├───boundary
+│   └───Vector data
+├───Geomorphic-Map
+│   └───Vector data
+├───Reef-Extent
+│   └───Vector data
+├───Turbidity-Q3-2023
+│   └───Raster data
+
```
Comment (ConnectedSystems) [inline README.md]:
```suggestion
│   └───Vector data
└───Turbidity-Q3-2023
    └───Raster data

```

File: README.md [hunk 3]
```diff
@@ -58,12 +64,33 @@ MPA_DATA_DIR
...
+├───Hs
+│   ├───Cairns-Cooktown
+│   ├───FarNorthern
+│   ├───Mackay-Capricorn
+│   └───Townsville-Whitsunday
+├───Tp
 │   ├───Cairns-Cooktown
 │   ├───FarNorthern
 │   ├───Mackay-Capricorn
 │   └───Townsville-Whitsunday
```
Comment (ConnectedSystems) [inline README.md]:
```suggestion
└───Tp
    ├───Cairns-Cooktown
    ├───FarNorthern
    ├───Mackay-Capricorn
    └───Townsville-Whitsunday
```

File: README.md [hunk 4]
```diff
@@ -131,6 +164,9 @@ Data Collection.
 https://doi.org/10.48610/8246441
 https://espace.library.uq.edu.au/view/UQ:8246441
 
+### ACA data
+https://www.allencoralatlas.org/
```
Comment (ConnectedSystems) [inline README.md]: Spacing is needed

```suggestion
### ACA data

https://www.allencoralatlas.org/
```

**Review by ConnectedSystems — CHANGES_REQUESTED** [review]
Minor formatting

File: README.md [hunk 5]
```diff
@@ -140,8 +176,16 @@ structures like reefs.
 
 Often this might be the only data available.
 
+#### Resolution
+
+`MPA raster data` (Bathymetry, Benthic, Geomorphic, Slope, Waves) : 10 x 10m pixel size
+
+`ACA raster data` (Bathymetry, Turbidity, Waves) : 10 x 10m pixel size
```
Comment (ryulippmann) [inline README.md]: Separate waves as standalone dataset, netCDF format

**Pull request merged**